pytumblr/__init__.py

```diff
@@ -488,6 +488,23 @@ def edit_post(self, blogname, **kwargs):
         valid_options = ['id'] + self._post_valid_options(kwargs.get('type', None))
         return self.send_api_request('post', url, kwargs, valid_options)
 
+    @validate_blogname
+    def notes(self, blogname, id, **kwargs):
+        """
+        Gets the notes
+
+        :param blogname: a string, the url of the blog that houses the post
+        :param id: a string, the id of the post.
+        :param mode: a string. Undocumented. Automatically added by tumblr but it's use is not yet known.
+        :param before_timestamp: a string, retreives data before this timestamp
+
+        :returns: a dict created from the JSON response
+        """
+        url = "/v2/blog/{}/notes".format(blogname)
+        valid_options = ['id', "mode", "before_timestamp"]
```

Comment:
Nitpick but can we stick to single or double quotes for all of the elements here? Thank you :)